src/pattern-library/components/patterns/TablePatterns.js

```diff
@@ -0,0 +1,125 @@
+import Library from '../Library';
+
+// "Fixture" for example table contents
+function ExampleTBody() {
+  return (
+    <tbody>
+      <tr>
+        <td>Alphanumeric Balloons</td>
+        <td>Champagne Delusions</td>
+      </tr>
+      <tr>
+        <td>Elephantine Fry-ups</td>
+        <td>Gargantuan Hiccups</td>
+      </tr>
+      <tr className="is-selected">
+        <td>Illicit Jugglers</td>
+        <td>Katydid Lozenges Meringue</td>
+      </tr>
+      <tr>
+        <td>Alphanumeric Balloons</td>
+        <td>Champagne Delusions</td>
+      </tr>
+      <tr>
+        <td>Elephantine Fry-ups</td>
+        <td>Gargantuan Hiccups</td>
+      </tr>
+      <tr>
+        <td>Illicit Jugglers</td>
+        <td>Katydid Lozenges Moebius</td>
+      </tr>
+      <tr>
+        <td>Elephantine Fry-ups</td>
+        <td>Gargantuan Hiccups</td>
+      </tr>
+      <tr>
+        <td>Illicit Jugglers</td>
+        <td>Katydid Lozenges Meringue</td>
+      </tr>
+      <tr>
+        <td>Alphanumeric Balloons</td>
+        <td>Champagne Delusions</td>
+      </tr>
+    </tbody>
+  );
+}
+
+export default function TablePatterns() {
+  return (
+    <Library.Page title="Tables">
+      <p>
+        These <code>table</code> patterns support a basic table layout that
+        adapts to available space. They are intended for simpler tabular
+        display: maximum 2 or possibly 3 columns. Remember that{' '}
+        <code>table</code> content needs to be usable in tight and narrow
+        spaces.
+      </p>
+      <Library.Pattern title="Table">
+        <Library.Example title="Basic table" variant="wide">
+          <p>
+            By default, a <code>table</code> will fill available horizontal
+            space, and will use whatever height is needed to render its rows.{' '}
+            <code>tr.is-selected</code> styles a row as selected.
+          </p>
+
+          <Library.Demo withSource>
+            <table className="hyp-table">
+              <thead>
+                <tr>
+                  <th scope="col">Column A</th>
+                  <th scope="col">Column B</th>
+                </tr>
+              </thead>
+              <ExampleTBody />
+            </table>
+          </Library.Demo>
+        </Library.Example>
+
+        <Library.Example title="Adjusting column widths">
+          <p>
+            Table column widths may be adjusted by styling <code>thead th</code>{' '}
+            elements. In this example, the column widths are set to 30% and 70%.
+          </p>
+          <Library.Demo withSource>
+            <table className="hyp-table">
+              <thead>
+                <tr>
+                  <th scope="col" style="width:30%">
```

Comment:
Ah! I also just TIL that particular detail. From the styling side, the `table-layout: fixed` rule in the SASS makes it so that the table and column width is derived from the first row in the table, making it possible to style the `th` widths and have that reliably apply to the whole table.